flows/engine/assets.go

```diff
@@ -0,0 +1,66 @@
+package engine
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/nyaruka/goflow/flows"
+	"github.com/nyaruka/goflow/flows/definition"
+)
+
+type assets struct {
```

Comment:
I think I don't mind assets too much. These play a similar role for the engine as sprites for a game, in a way. I think my expectation though is that they are static. While these are static within a single execution cycle, they are likely to change quite a bit. That's probably okay though.